README.md

```diff
@@ -1,4 +1,4 @@
-# Astrobee Robot Software
+# Astrobee Robot Software v1
```

Comment:
Fix not maintainable, move to a constant

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Not maintainable

Reply:
Looks good.  Now maintainable.

---
**Sema Reaction:** :ok_hand: This code looks good | **Sema Tags:** Maintainable